### PR TITLE
Enable debounce at 5 percent on nightly

### PR DESCRIPTION
**Pull request by pilgrim-brave**
Feature is described here: https://github.com/brave/brave-browser/issues/15090

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -1114,6 +1114,33 @@
...
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": ["NIGHTLY"],
+                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+            }
```
Comment (moritzhaller) [inline seed/seed.json:1143]: maybe add min version, analogous to e.g. https://github.com/brave/brave-variations/pull/125/files#diff-7ac2945ede68c73330363911678f6c9a2a6290b70e2dd170868295f65da7e872R1113

**Review by moritzhaller — APPROVED** [review]

File: seed/seed.json (hunk 1 above)
Comment (pilgrim-brave) [reply seed/seed.json:1144]: added in https://github.com/brave/brave-variations/commit/8a08e3cb52f8f5b37f3b3f6908dad9fd1d794593

File: seed/seed.json [hunk 2]
```diff
@@ -1114,6 +1114,34 @@
...
+                        "disable_feature": ["BraveDebounce"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "94.0.4606.54",
```
Comment (iefremov) [inline seed/seed.json:1141]: @pilgrim-brave @moritzhaller this is a wrong version format, see other entries

Comment (pilgrim-brave) [reply seed/seed.json:1141]: Well I see them. Is there a way to get the version in that format from, say, a Brave Nightly build?

Comment (iefremov) [reply seed/seed.json:1141]: it is shown on `brave://system` for example. Or you manually craft it by taking chromium major version (aka `94`) and appending brave version (e.g. `1.31.55`)

**Pull request merged**